src/bindings/meson.build

```diff
@@ -18,6 +18,10 @@ if (bindings.contains('cxx') == false and bindings.contains('mono'))
   )
 endif
 
+if bindings.contains('lua') and not have_elua
```

Comment:
Where does this `have_elua` comes from?

Reply:
[here](https://github.com/expertisesolutions/efl/blob/devs/expertise/native-windows/meson.build#L383v) you are, sir

Reply:
Lovely :D Thanks